tests.sh

```diff
@@ -3,22 +3,62 @@
 #Run daos tests like IOR/MDtest and self_test(cart)
 #----------------------------------------------------
 
-#Unload modules that are not needed on Frontera
-module unload impi pmix hwloc intel
-module list
-
-#Parameter to be updated for each sbatch
+# Configurable parameters to be updated for each sbatch
 DAOS_AGENT_DRPC_DIR="/tmp/daos_agent"
 ACCESS_PORT=10001
-MPI="mvapich2" #supports mvapich2, openmpi, or mpich
+MPI_TARGET="mvapich2" #supports mvapich2, openmpi, or mpich
 OMPI_PARAM="--mca oob ^ud --mca btl self,tcp --mca pml ob1"
 
-if [ "$MPI" != "mvapich2" ] && [ "$MPI" != "openmpi" ] && [ "$MPI" != "mpich" ]; then
-    echo "Unknown MPI. Please specify either mvapich2, openmpi, or mpich"
+if [ "${MPI_TARGET}" != "mvapich2" ] &&
+   [ "${MPI_TARGET}" != "openmpi" ] &&
+   [ "${MPI_TARGET}" != "mpich" ]; then
+    echo "Unknown MPI_TARGET. Please specify either mvapich2, openmpi, or mpich"
     exit 1
 fi
 
-#Others
+# Set default value for a variable that is undefined/empty
+function set_default(){
+    local var_name=${1}
+    local var_default=${2}
+
+    if [ -z "${!var_name}" ]; then
+        eval ${var_name}=\${var_default}
+    fi
+}
+
+# Set undefined/default test params
+set_default NUMBER_OF_POOLS 1
+set_default no_of_ps $(($DAOS_CLIENTS * $PPC))
+
+# Print all relevant test params / env variables
+echo "TESTCASE        : ${TESTCASE}"
+echo "DAOS_SERVERS    : ${DAOS_SERVERS}"
+echo "DAOS_CLIENTS    : ${DAOS_CLIENTS}"
+echo "PPC             : ${PPC}"
+echo "RANKS           : ${no_of_ps}"
+echo "OCLASS          : ${OCLASS}"
+echo "DIR_OCLASS      : ${DIR_OCLASS}"
+echo "SW_TIME         : ${SW_TIME}"
+echo "N_FILE          : ${N_FILE}"
+echo "NUM_POOLS       : ${NUMBER_OF_POOLS}"
+echo "POOL_SIZE       : ${POOL_SIZE}"
+echo "CHUNK_SIZE      : ${CHUNK_SIZE}"
+echo "BYTES_READ      : ${BYTES_READ}"
+echo "BYTES_WRITE     : ${BYTES_WRITE}"
+echo "SLURM_JOB_ID    : ${SLURM_JOB_ID}"
+echo "SEGMENTS        : ${SEGMENTS}"
+echo "XFER_SIZE       : ${XFER_SIZE}"
+echo "BLOCK_SIZE      : ${BLOCK_SIZE}"
+echo "FPP             : ${FPP}"
+echo "CONT_RF         : ${CONT_RF}"
+echo "MPI_TARGET      : ${MPI_TARGET}"
+
```

Comment:
This is where we can print any test params that can be parsed later for the result CSV